### ISSUE TITLE
extend Good Boy License with the MIT license?

### ISSUE THREAD

**Issue by philkunz** [icons8/windows-10-icons#2]
any chance you could extend your Good Boy license with the MIT license, so we have a common, reliable license?

**Comment by JD-Robbs** [comment]
Yes, that would be nice. Much like it was done for [Flat Color Icons](https://github.com/icons8/flat-color-icons).

**Comment by snlacks** [comment]
Yes, I really want to get this as a tattoo or use it in an app without having to fight "legal;" mom wouldn't like the latter, but she'd be okay with the former.

**Comment by radioacoustick** [comment]
Yes, that would be nice. We are still waiting for decision.

**Comment by winnieXY** [comment]
Debian Developer here. 

Please provide a standard open source license together with your icon pack if possible. The good boy license is _not_ compatible with any open source license and prevents all usage of the icons in open source projects. 

A license under either good boy or MIT as e.g. in your  flat-color-icons  repository would be greatly appreciated.

**Comment by snlacks** [comment]
The license in the package.json is CC-BY-4.0, Creative Commons. The License files seems like a joke. If we opened a PR to change the license file would it get merged? We could preserve the joke license in the readme or license history or something.

**Comment by winnieXY** [comment]
that would be great :-) I'm not sure if any of the original developers/authors is following here - currently the repository looks quite dead in my eyes unfortunately.

**Comment by flagman** [comment]
We hear you :) 
It’s now dual-licensed.
